client/state/sites/vouchers/actions.js

```diff
@@ -75,6 +79,68 @@ export const vouchersRequestFailureAction = ( siteId, error ) => {
 	};
 };
 
+export const vouchersAssignReceiveAction = ( siteId, serviceType, voucher ) => {
+	return {
+		type: SITE_VOUCHERS_ASSIGN_RECEIVE,
+		siteId,
+		serviceType,
+		voucher
+	};
+};
+
+/**
+ * Action creator function
+ *
+ * Return SITE_VOUCHERS_ASSIGN_REQUEST action object
```

Comment:
These comments aren't too helpful in my opinion. It's pretty obvious that they are action creator functions already because that's the file they are in and it's also pretty clear what they return, as it's found a few lines down.

Could we describe _why_ they are here maybe in a bit more detail instead of describing _what_ they do, which is pretty easy to figure out?

> Indicates that we are making an API request to request voucher information

Something like that ^^^. Of course, please don't use that because I doubt it describes what this function is for, which is part of the issue - I don't know why this function exists or when I should use it.